extensions-contrib/grpc-query/pom.xml

```diff
@@ -235,6 +242,26 @@
           </execution>
         </executions>
       </plugin>
+        <plugin>
+            <groupId>org.xolstice.maven.plugins</groupId>
+            <artifactId>protobuf-maven-plugin</artifactId>
+            <version>0.6.1</version>
+            <configuration>
+                <protocArtifact>com.google.protobuf:protoc:3.21.7:exe:${os.detected.classifier}</protocArtifact>
+                <pluginId>grpc-java</pluginId>
+                <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.52.0:exe:${os.detected.classifier}</pluginArtifact>
+            </configuration>
```

Comment:
As it turns out, we can't do the compile in this project. The generated code contains references to Guava. To ensure those references point to our shaded Guava, we have to do the compile in the `grpc_shaded` project.

Reply:
Ok, I will move the test .proto file to the grpc_shaded project

Reply:
We discussed this elsewhere. It may be that it is only the gRPC parts of the .proto files that depend on (the shaded) Guava, and that simple protobufs only depend on the (unshaded) Protobuf classes. If so, let's go ahead and leave the code here.